tooling/bundler/src/bundle/windows/templates/install-task.ps1

```diff
@@ -0,0 +1,25 @@
+# Adapted from https://superuser.com/a/532109
```

Comment:
This is tricky, because of licensing. Also, all code files need spdf license headers.

Reply:
I updated the header with the SPDX License. If it doesn't cover the issue correctly though, I'm happy to investigate other options as well.